core/src/Log.purs

```diff
@@ -183,25 +182,18 @@ justOrDieWith' value msg = case value of
     die' msg
 
 rightOrDie :: ∀ b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable err => Either err x -> m x
-rightOrDie value = rightOrDieWith value identity
+rightOrDie = rightOrDieWith identity
 
 rightOrDie_ :: ∀ b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable err => Either err x -> m Unit
 rightOrDie_ = void <<< rightOrDie
 
-rightOrDieWith :: ∀ a b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable a => Either err x -> (err -> a) -> m x
-rightOrDieWith value toMsg = case value of
+rightOrDieWith :: ∀ a b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable a => (err -> a) -> Either err x -> m x
+rightOrDieWith toMsg value = case value of
   Right a ->
     pure a
   Left err ->
     die $ toMsg err
 
-rightOrDieWith' :: ∀ a b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable a => Either err x -> (err -> Array a) -> m x
-rightOrDieWith' value toMsg = case value of
```

Comment:
This was also not used anywhere, and also is redundant, since there is an instance `Loggable a => Loggable (Array a)`